network/p2p/keyutils/keyTranslator_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"math"
 	"testing"
 
-	"github.com/btcsuite/btcd/btcec"
+	btcec "github.com/btcsuite/btcd/btcec/v2"
```

Comment:
@tarakby this should be safe, but just in case tagging you.

Reply:
Thanks for the flag. I had a look and all looks good 👌🏼